src/mainboard/clevo/adl-p/variants/nv40pz/romstage.c

```diff
@@ -21,5 +25,39 @@ void mainboard_memory_init_params(FSPM_UPD *mupd)
 	mupd->FspmConfig.DmiMaxLinkSpeed = 4;
 	mupd->FspmConfig.GpioOverride = 0;
 
+/* Use pre-processor because CONFIG_INTEL_TXT_CBFS_BIOS_ACM is not defined otherwise */
+#if CONFIG(INTEL_TXT)
+	size_t acm_size = 0;
+	uintptr_t acm_base;
+
+	if (CONFIG(INTEL_TXT)) {
```

Comment:
This `if` can be dropped.

Reply:
https://github.com/Dasharo/coreboot/pull/685/commits/507f28e0b9ba3f8225b7d9b18150effd3f76065d